### DISCUSSION TITLE
Prev Color and Next Color and more

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#119] [General]
![colors34](https://github.com/user-attachments/assets/75f73209-6207-48eb-b7b5-477a8dc0e848)

New commands to cycle between colors.

For instance from red -> green -> blue -> yellow -> none -> red

In either direction.

---

Also added more shift+mousewheel settings, for favorites, footer, title.

![mwshele](https://github.com/user-attachments/assets/a9071c8e-7821-4014-a80c-d4fc9e5b52cd)

---

Also in commands replaced instances of `Previous` to `Prev`.

![prev234](https://github.com/user-attachments/assets/b2a9b0f6-fe03-4467-9b22-4cf5a4ab246e)

It's easier to find stuff.

---

![howverqw](https://github.com/user-attachments/assets/eaf3e8e5-9fb9-45a1-98a5-89ea91a4edf9)

New `Wheel Hover Item` setting, enabled by default.

When you do shift+wheel when hovering an item, that item is used as the `item` instead of the selected one.

Unless that setting is disabled.

---

Also the Colors setting icon was changed from Zimbabwe's flag to Burundi's flag: 🇧🇮

**Comment by N3C2L** [comment]
Very useful, thanks!
